db.go

```diff
@@ -20,7 +20,9 @@ const maxMmapStep = 1 << 30 // 1GB
 const version = 2
 
 // Represents a marker value to indicate that a file is a Bolt DB.
+// TODO(ptab): Merge the magic.
 const magic uint32 = 0xED0CDAED
+const Magic uint32 = 0xED0CDAED
```

Comment:
The Magic should be invisible to users/client applications.

Reply:
Addressed in the other PR: https://github.com/etcd-io/bbolt/pull/354

In general we are missing an internal library that bbolt cli can use and share with the core implementation instead of having copy-paste of the data-structures and constants.